bin/bundle.cmd

```diff
@@ -0,0 +1,117 @@
+@ruby -x "%~f0" %*
+@exit /b %ERRORLEVEL%
+
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+#
+# This file was generated by Bundler.
+#
+# The application 'bundle' is installed as part of a gem, and
+# this file is here to facilitate running it.
+#
+
+require "rubygems"
+
+m = Module.new do
+  module_function
```

Comment:
same here, What is used if this file, why you pushed it

Reply:
same as all bin/ folder files any reason to push or you're missing adding git ignore file
NOTE: understand use of `gitignore`